### DISCUSSION TITLE
@types/chai-jwt library is missing

### DISCUSSION THREAD

**Discussion by Mona-Superscientist** [DefinitelyTyped/DefinitelyTyped#55284] [Request a new @types package] [answered]
We need to provide @type library for `chai-jwt` plugin https://www.npmjs.com/package/chai-jwt

**Comment by kasmacioma** [comment, marked as answer]
Done 🎉
Check it out [here](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/chai-jwt) or [on npm](https://www.npmjs.com/package/@types/chai-jwt).
